_policy/referees.md

```diff
@@ -1,15 +1,448 @@
+    Commission Members taking into account the requirement for a cost efficient and effective
+    process of appointment to be used.  This could include (but should not be limited to) an
+    appointment being made on the basis of application only without interview or an interview via a
+    telecommunications method.
+7.  Application for the above position will be advertised on the FIT website. All policy documents
+    will be available on the FIT website.
+
+### FIT Director of Referees Selection Criteria
+
+The following are selection criteria for the FIT Referee Director:
+
+1.  Would be expected to have coached at National / High Performance level for minimum five (5)
+    years including significant exposure to selection and coaching of international referees, as
+    well as having observed and coached at international events.
+2.  Demonstrates a capacity to exercise mature judgment and participate in impartial assessment of
+    Referees from all Member NTA's.
+3.  Possess maturity, self confidence and tact, and the ability to deal effectively with Referees,
+    Players and Officials at all levels.
+4.  Be available to undertake courses, seminars, lectures etc, so as to be able understand the needs
+    of Member countries.   Implement strategies for the development of referees and referee coaches,
+    globally as per the FIT Strategic Plan and Priorities..
+5.  Implement strategies for the implementation and ongoing development of referee personnel,
+    globally as per the FIT Strategic Plan and Priorities.
+6.  Must possess knowledge of the standards and behaviour expected of a FIT official and or
+    representative, and an acceptance of these standards as set out in the FIT Code of Conduct i.e. a
+    reference point for the standards.
+7.  Must possess a minimum of an Advanced Elite Certificate such as:
+8.  TFA Level 4 Referees Coaching Certificate;
+9.  TNZ Level 4 Referees Coaching Certificate; or 
+10. An accepted recognized equivalent within the Member NTA.
+11. Possess the ability to communicate effectively with a national or international body of referees
+    to update on current issues, trends and to disseminate information.
+12. Constantly revise the delivery of Referee Badge Level Courses and Referee Coaching Courses for
+    distribution globally.
+13. Regularly update all Member NTA's with any advancement in referee coaching techniques and/or
+    trends.
+14. Coordinate FIT Referee Commission Member visits and expected outcomes to Member NTA's.
+15. Implement a succession plan to develop referees and referee coaches, globally.
+16. Is a volunteer and not in the employment of FIT or any Member NTA.
+17. Must be non active as a referee (unless the appointment is to an Associate position for
+    developmental purposes.
+18. An outline role description for the FIT Director of Referees is attached as Appendix 1.
+
+### FIT Referee Commission Member Appointment & Criteria
```

Comment:
Likewise, is this better removed and dealt with in the [Referees Commission Member](http://docs.internationaltouch.org/position/referees-commission/member) position description?